app/models/employment.rb

```diff
@@ -22,6 +36,7 @@ def duplicate
   end
 
   def current_role?
-    current_role == "yes"
+    current_role.present? && current_role != "no"
```

Comment:
[Minor] Probably something I am missing here but using current_role == "yes" here makes more sense to me as that's the logic we are using to set is_current_role on line 18.

Reply:
again, reverting